test/live-pixel.js

```diff
@@ -19,4 +35,83 @@ describe('LivePixel', function () {
         let locationHeader = result.headers.location
         expect(locationHeader).to.include('n3pc=true', 'Failed to verify n3pc=true in redirect')
     })
+
+    it('Test LivePixel record from the Stream has the autogenerated properties set', async function () {
+        this.timeout(90000)
+        let lpUrl = lpData.paramsWithCookieSet()
+        let cookie = lpUrl.networkUserId
+
+        const p = client.getData(recordPredicate(cookie))
```

Comment:
`p` is quite a short and undescriptive name
```suggestion
        const p = client.getData(recordPredicate(cookie))
```

Reply:
renamed